### ISSUE TITLE
Create install_all.sh script

### ISSUE THREAD

**Issue by kurtwheeler** [AlexsLemonade/refinebio#714]
### Context

Right now we have a decently long getting started section. It's broken up like this so that we don't make a ton of changes to the user's computer without their knowledge.

### Problem or idea

Everything we instruct the user to do could be done by a single script. We should offer this as an alternative to walking through our guide. This also will give us the ability to test that our system runs after following our instructions exactly.

### Solution or next step

Create an `install_all.sh` script that will detect whether it's installing onto Mac or Ubuntu, install any missing dependencies, start the database, build local images, get Nomad running, and anything else that should be done before a developer can run the entire system locally.

### New Issue Checklist

- [x] The title is short and descriptive.
- [x] You have explained the context that led you to write this issue.
- [x] You have reported a problem or idea.
- [x] You have proposed a solution or next step.
